modules/apps/frontend-js/frontend-js-spa-web/src/main/resources/META-INF/resources/screen/RequestScreen.js

```diff
@@ -282,20 +286,23 @@ class RequestScreen extends Screen {
 				}),
 			new Promise((_, reject) => {
 				setTimeout(
-					() => reject(new Error(REQUEST_TIMEOUT)),
+					() => reject(new Error(REQUEST_TIMEOUT_MSG)),
 					this.timeout
 				);
 			}),
 		]).catch((reason) => {
 			switch (reason.message) {
-				case REQUEST_TIMEOUT:
+				case REQUEST_TIMEOUT_MSG:
 					reason.timeout = true;
 					break;
-				case REQUEST_PREMATURE_TERMINATION:
+				case REQUEST_PREMATURE_TERMINATION_MSG:
+				case FAILED_TO_FETCH_MSG:
+				case NETWORK_ERROR_MSG:
+				case PREFLIGHT_ERROR_MSG:
```

Comment:
I'm amazed you got away without sorting this.

Reply:
I swear I sorted it... in some version of the PR 😂